Annex60/Fluid/Sensors/TraceSubstancesPpm.mo

```diff
@@ -0,0 +1,69 @@
+within Annex60.Fluid.Sensors;
+model TraceSubstancesPpm
+  "Ideal one port trace substances sensor outputting in parts per million"
+  extends Annex60.Fluid.Sensors.BaseClasses.PartialAbsoluteSensor;
+  extends Modelica.Icons.RotationalSensor;
+  parameter String substanceName = "CO2" "Name of trace substance";
+
+  parameter Real MM_fraction = 44/29
```

Comment:
@Mathadon : This should by 29/44, not 44/29. I will correct it and add a regression test.